src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -66,7 +67,13 @@ public class ExceptionlessSink : ILogEventSink {
         public ExceptionlessSink(Func<EventBuilder, EventBuilder> additionalOperation = null, bool includeProperties = true, ExceptionlessClient client = null) {
             _additionalOperation = additionalOperation;
             _includeProperties = includeProperties;
-            _client = client ?? ExceptionlessClient.Default;
+            if (client != null) {
+                _client = client;
+            }
+            else {
+                _client = ExceptionlessClient.Default;
+                _client.Configuration.UseLogger(new SelfLogLogger());
```

Comment:
So you could do a type check and just set it and simplify the code a bit too to do the previous `_client = client ?? ExceptionlessClient.Default;`